vllm/model_executor/input_metadata.py

```diff
@@ -30,10 +44,19 @@ def __init__(
     ) -> None:
         self.is_prompt = is_prompt
         self.prompt_lens = prompt_lens
+        self.num_prompt_tokens = num_prompt_tokens
+        self.num_generation_tokens = num_generation_tokens
         self.max_seq_len = max_seq_len
         self.start_loc = start_loc
         self.max_context_len = max_context_len
         self.slot_mapping = slot_mapping
+        # Index: The batched sequence's index.
+        # Value: The length of attention context.
+        # NOTE(sang): When it is prefill/decoding,
+        # the definition is different. For prefill,
+        # it means the the length of KV that are cached
+        # excluding the new KVs. In decoding, this
+        # includes a new KV.
```

Comment:
Could you fit these comments again for the 80-character line rule? It seems the lines here are too short.